Documentation/Settings.cfg

```diff
@@ -34,3 +34,4 @@ h2document     = https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-u
 t3coreapi     = https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/
 t3tca         = https://docs.typo3.org/m/typo3/reference-tca/master/en-us/
 t3tsref       = https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/
+t3admpanel    = https://docs.typo3.org/c/typo3/cms-adminpanel/master/en-us/
```

Comment:
Can we use `admin` instead of `adm`? :pray:

Reply:
I spose. I think that's the actual extension name, or key? I was just trying to be proper.

Reply:
I'm happy for you to go ahead and change it. I really don't think it's a big deal here, because this is something that only the doc maintainers will see. And I personally think it's fairly obvious.